nodes/parachain/src/command.rs

```diff
@@ -336,6 +336,16 @@ pub fn run() -> Result<()> {
 					)?;
 					cmd.run(partials.client)
 				}),
+				#[cfg(not(feature = "runtime-benchmarks"))]
+				(BenchmarkCmd::Storage(_), _) => {
+					return Err(sc_cli::Error::Input(
```

Comment:
Clippy noticed the return statement is unneeded.

Reply:
91d932b5945b536d9557524aa4619ed5af0912ae